packages/protobuf-bench/src/gen/protobuf-es/buf/alpha/image/v1/image_pb.ts

```diff
@@ -17,7 +17,7 @@
 /* eslint-disable */
 
 import type {BinaryReadOptions, FieldList, JsonReadOptions, JsonValue, PartialMessage, PlainMessage} from "@bufbuild/protobuf";
-import {DescriptorProto, EnumDescriptorProto, FieldDescriptorProto, FileOptions, Message, ServiceDescriptorProto, SourceCodeInfo, proto2} from "@bufbuild/protobuf";
+import {DescriptorProto, EnumDescriptorProto, FieldDescriptorProto, FileOptions, Message, proto2, ServiceDescriptorProto, SourceCodeInfo} from "@bufbuild/protobuf";
```

Comment:
Actually tried to port the Go sort. ECMAScript defines a `caseFirst` option in Intl.Collator, but it's unimplemented in Node and Chrome.